### PR TITLE
BUGFIX: Invalid import in main.js

### PR DESCRIPTION
**Pull request by yauheni-chaburanau-it**
So basically I faced this problem:
```
Uncaught Exception:
TypeError: app.requestSingleInstanceLock is not a function
    at Object.<anonymous> (/Users/user/.config/test/data/test/vendor/astilectron/main.js:9:35)
    at Module._compile (internal/modules/cjs/loader.js:880:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:892:10)
    at Module.load (internal/modules/cjs/loader.js:735:32)
    at Module._load (internal/modules/cjs/loader.js:648:12)
    at Module._load (electron/js2c/asar.js:717:26)
    at Function.Module._load (electron/js2c/asar.js:717:26)
    at loadApplicationPackage (/Users/user/.config/test/data/test/vendor/electron-darwin-amd64/test.app/Contents/Resources/default_app.asar/main.js:109:16)
    at Object.<anonymous> (/Users/user/.config/test/data/test/vendor/electron-darwin-amd64/test.app/Contents/Resources/default_app.asar/main.js:155:9)
    at Module._compile (internal/modules/cjs/loader.js:880:30)
```

As I understand the problem is in import definition in `main.js`

I'm using this library together with `go-astilectron` library and this error happens only if 
```go
astilectron.Options{
	...
	SingleInstance:     true,
	...
}
```

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const app = require("electron");
+const app = require("electron").app;
```
Comment (asticode) [inline main.js]: I'd rather use

```javascript
const { app } = require("electron")
```

**Review by asticode — CHANGES_REQUESTED** [review]
Nice catch !

Minor change needed though.

Cheers

File: main.js (hunk 1 above)
Comment (yauheni-chaburanau-it) [reply main.js]: I'm not so familiar with JS, but I did what I did just to be consistent with the imports below.
Should I update those imports as well?

Comment (asticode) [reply main.js]: yeah you should be able to do something like

```javascript
const { start, lastWindow } = require("./index")
```

Comment (yauheni-chaburanau-it) [reply main.js]: Done
Could you please check

**Comment by asticode** [comment]
LGTM

**Comment by asticode** [comment]
FYI I've tagged `astilectron` with `v0.34.0` and bumped both `go-astilectron` and `go-astilectron-demo`

**Comment by yauheni-chaburanau-it** [comment]
> FYI I've tagged `astilectron` with `v0.34.0` and bumped both `go-astilectron` and `go-astilectron-demo`

Thank you very much

**Pull request merged**